components/style/servo/selector_parser.rs

```diff
@@ -100,44 +100,55 @@ impl PseudoElement {
 #[cfg_attr(feature = "servo", derive(HeapSizeOf))]
 #[allow(missing_docs)]
 pub enum NonTSPseudoClass {
-    AnyLink,
-    Link,
-    Visited,
     Active,
+    AnyLink,
+    Checked,
+    Disabled,
+    Enabled,
     Focus,
     Fullscreen,
     Hover,
-    Enabled,
-    Disabled,
-    Checked,
     Indeterminate,
-    ServoNonZeroBorder,
+    Lang(String),
```

Comment:
How does it relate? It's effectively an immutable string, with `Box<str>` you can deref to `&str` and still use `split`, etc.